test/e2e/network_policy.go

```diff
@@ -41,33 +41,34 @@ failed or succesfully connected as expected.
 var _ = framework.KubeDescribe("NetworkPolicy", func() {
 	f := framework.NewDefaultFramework("network-policy")
 
-	It("should support setting DefaultDeny namespace policy [Feature:NetworkPolicy]", func() {
+	It("should support a 'default-deny' policy [Feature:NetworkPolicy]", func() {
 		ns := f.Namespace
 
 		By("Create a simple server.")
 		podServer, service := createServerPodAndService(f, ns, "server", []int{80})
-		defer func() {
-			By("Cleaning up the server.")
-			if err := f.ClientSet.Core().Pods(ns.Name).Delete(podServer.Name, nil); err != nil {
-				framework.Failf("unable to cleanup pod %v: %v", podServer.Name, err)
-			}
-		}()
-		defer func() {
-			By("Cleaning up the server's service.")
-			if err := f.ClientSet.Core().Services(ns.Name).Delete(service.Name, nil); err != nil {
-				framework.Failf("unable to cleanup svc %v: %v", service.Name, err)
-			}
-		}()
+		defer cleanupServerPodAndService(f, podServer, service)
 		framework.Logf("Waiting for Server to come up.")
 		err := framework.WaitForPodRunningInNamespace(f.ClientSet, podServer)
 		Expect(err).NotTo(HaveOccurred())
 
 		// Create a pod with name 'client-a', which should be able to communicate with server.
```

Comment:
nit: the comment says pod `'client-a'` but it's `'client-can-connect'` in reality (same for a few other comments)